zuul.d/jobs.yaml

```diff
@@ -408,8 +408,9 @@
     description: Run a functional test against jammy-yoga
     parent: func-target-pre-jobs
     dependencies:
-      - name: tox-py310
```

Comment:
Removing this soft dependency is necessary, if we remove the py310 job from the template, then it won't be run, and it will be ignored here. Removing it here, though, means we can't even opt-in to having it block the functional runner when we're ready for it